functions.php

```diff
@@ -13,11 +13,11 @@ function createDBConnection(): PDO {
 /**
  * A query to the database that fetches the data in all the fields and creates a recipes assoc array
  *
- * @param $db
+ * @param PDO $db
  * @return array
  */
-    function fetchAllRecipes($db): array {
-        $query = $db->prepare('SELECT `recipe`, `cuisine`, `time`, `link` FROM `recipes`;');
+    function fetchAllRecipes(PDO $db): array {
+        $query = $db->prepare("SELECT `recipe`, `cuisine`, `time`, `link` FROM `recipes` WHERE `deleted` = '0';");
```

Comment:
Can use single quotes here and have the 0 out of quotes. Personal preference though.